inscriptions/ISic001185.xml

```diff
@@ -190,15 +195,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140669">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MH">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="omitted">Ἀρ</supplied>χεδάμου.
-                    <lb n="2"/>
+                    <lb n="1"/><supplied reason="omitted">Ἀρ</supplied>χεδάμου.
```

Comment:
actually, just write this out as Ἀρχεδάμου without any tags (ideally delete the final full-stop also); and then report the text as you do in the apparatus (But, for 'Gualt' write 'Gualtherus' in full)